src/app/pages/pages.routing.ts

```diff
@@ -0,0 +1,27 @@
+import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
import { NgModule } from "@angular/core";
```